tests/test_sentences_endpoints.py

```diff
@@ -0,0 +1,127 @@
+    return loop.run_until_complete(sanic_client(app))
+
+
+@pytest.fixture
+def generateData(request):
+    # adding dummy sentences
+    insertDummySentences()
+
+    def fin():
+        removeCollections()
+
+    request.addfinalizer(fin)
+
+
+async def test_getSentences_with_noParam_shouldPass(test_cli, generateData):
+    resp = await test_cli.get('/sentences')
+    resp_json = await resp.json()
+    assert resp_json["total_results"] == 10
+    assert len(resp_json["results"]) == 10
+    assert resp_json["page"] == 1
+    assert resp_json["results"][0]["language"]["name"] == "francais"
+    assert resp.status == 200
+
+
+async def test_getSentencesById_with_unKnownID_shouldFailed(test_cli, generateData):
+    resp = await test_cli.get('/sentences/veryunknown')
+    resp_json = await resp.json()
+    assert resp.status == 404
+    assert resp_json["message"] == "Object Not found"
+
+
+async def test_getSentences_with_bothExpectedParam_shouldPass(test_cli, generateData):
+    params = {'page': 2, 'language': 'francais'}
+    resp = await test_cli.get('/sentences', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp.status == 200
+    assert resp_json["total_results"] == 6
+    assert len(resp_json["results"]) == 0
+    assert resp_json["page"] == 2
+
+
+async def test_getSentences_with_validPageNum_shouldPass(test_cli, generateData):
+    params = {'page': 20}
+    resp = await test_cli.get('/sentences', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp.status == 200
+    assert resp_json["total_results"] == 10
+    assert len(resp_json["results"]) == 0
+    assert resp_json["page"] == 20
+
+
+async def test_getSentences_with_invalidPageNum_shouldFail(test_cli, generateData):
+    params = {'page': "test"}
+    resp = await test_cli.get('/sentences', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp_json["message"] == "Required Integer, found String"
+    assert resp.status == 400
+
+
+async def test_getSentences_with_validLanguage_shouldPass(test_cli, generateData):
+    params = {'language': "francais"}
+    resp = await test_cli.get('/sentences', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp.status == 200
+    assert resp_json["total_results"] == 6
+    assert len(resp_json["results"]) == 6
+    assert resp_json["results"][0]["language"]["name"] == "francais"
+
+
+async def test_addSentences_with_noParam_shouldFail(test_cli):
+    resp = await test_cli.post('/sentences')
+    resp_json = await resp.json()
+    assert resp_json["message"] == "Invalid Payload."
+    assert resp.status == 400
+
+
+async def test_addSentences_with_unexpectedParam_shouldFail(test_cli):
+    params = {"invalid_field": 2, "language": "dummy"}
+    resp = await test_cli.post('/sentences', data=json.dumps(params))
+    resp_json = await resp.json()
+    assert resp.status == 400
+    assert resp_json["message"] == "Invalid Payload."
```

Comment:
Invalid request body seems more accurate to me, but this is fine.

Reply:
okay